must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -67,3 +67,18 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+#Adding ready OCS nodes beforehand
+nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers|awk '{print $1}')
```

Comment:
Again here, do we really want to loop through *all* OCS labelled nodes, or just the Ready ones?

Reply:
Well, in my testing until yesterday, adding `/\yReady\y/{print $1}` was not printing anything and I don't know why, So I made the changes as it was without the `Ready` in awk command. But today it worked, not sure what was the reason, but it works for sure and I have changed the above line as well.